clients/da-vinci-client/src/main/java/com/linkedin/davinci/ingestion/isolated/IsolatedIngestionServer.java

```diff
@@ -673,10 +712,11 @@ private void initializeIsolatedIngestionServer() {
     LOGGER.info(
         "Starting report client with target application port: {}",
         configLoader.getVeniceServerConfig().getIngestionApplicationPort());
-    // Create Netty client to report status back to application.
+    // Create Netty client to report ingestion status back to main process.
```

Comment:
These are AHAC based client, not Netty client, right?

Reply:
Yes, basically same as we used in other place in our codebase.